### PR TITLE
DHFPROD-7374: Fix fullCycleTests

### PR DESCRIPTION
**Pull request by akshaysonvane**
#### Checklist: 
```diff
- Note: do not change the below
```

-  ##### Owner:

- [x] JIRA_ID included in all the commit messages
- [x] PR title is in the format JIRA_ID:Title
- [x] Rebase the branch with upstream
- [x] Squashed all commits into a single commit
- [N/A] Code passes ESLint tests
- [N/A] Added Tests
  

- ##### Reviewer:

- [N/A] Reviewed Tests

### CODE REVIEW

**Review by ryanjdew — DISMISSED** [review]

File: installer-for-dhs/src/main/java/com/marklogic/hub/dhs/installer/command/InstallIntoDhsCommand.java [hunk 1]
```diff
@@ -76,7 +77,9 @@ public void run(ApplicationContext context, Options options) {
         );
         commands.add(deployRolesCommand);
 
-        commands.add(new DeployAmpsCommand());
+        DeployAmpsCommand deployAmpsCommand = new DeployHubAmpsCommand();
+        deployAmpsCommand.setExecuteSortOrder(390);
```
Comment (srinathgit) [inline installer-for-dhs/src/main/java/com/marklogic/hub/dhs/installer/command/InstallIntoDhsCommand.java:81]: We always install dhs using  admin user. Not sure if it's needed.

File: marklogic-data-hub/src/main/java/com/marklogic/hub/dhs/installer/deploy/DeployHubAmpsCommand.java [hunk 2]
```diff
@@ -18,6 +18,15 @@
  */
 public class DeployHubAmpsCommand extends DeployAmpsCommand {
 
+    /**
+     * DeployHubAmpsCommand should run before we run LoadHubModulesCommand as we require the amp for getting the default
+     * rewriter to be present so that it can be used to generate the custom rewriter.
+     */
+   public DeployHubAmpsCommand() {
+       super();
+       setExecuteSortOrder(390);
```
Comment (srinathgit) [inline marklogic-data-hub/src/main/java/com/marklogic/hub/dhs/installer/deploy/DeployHubAmpsCommand.java:27]: Rob filed https://github.com/marklogic-community/ml-app-deployer/issues/432 . We may not need this if there is a new release.

File: installer-for-dhs/src/main/java/com/marklogic/hub/dhs/installer/command/InstallIntoDhsCommand.java (hunk 1 above)
Comment (akshaysonvane) [reply installer-for-dhs/src/main/java/com/marklogic/hub/dhs/installer/command/InstallIntoDhsCommand.java:81]: I don't think we should rely on the admin roles capability when we know that the real issue is with task prioritization.

File: marklogic-data-hub/src/main/java/com/marklogic/hub/dhs/installer/deploy/DeployHubAmpsCommand.java (hunk 2 above)
Comment (akshaysonvane) [reply marklogic-data-hub/src/main/java/com/marklogic/hub/dhs/installer/deploy/DeployHubAmpsCommand.java:27]: Yeah, I did talk to @rjrudin about this. This is an interim fix for now.

File: installer-for-dhs/src/main/java/com/marklogic/hub/dhs/installer/command/InstallIntoDhsCommand.java (hunk 1 above)
Comment (rjrudin) [reply installer-for-dhs/src/main/java/com/marklogic/hub/dhs/installer/command/InstallIntoDhsCommand.java:81]: Let's still just use DeployAmpsCommand here though, not DeployHubAmpsCommand, as the latter is specifically for a limited user.

**Review by rjrudin — CHANGES_REQUESTED** [review]
Looks good, just one tweak

File: installer-for-dhs/src/main/java/com/marklogic/hub/dhs/installer/command/InstallIntoDhsCommand.java (hunk 1 above)
Comment (akshaysonvane) [reply installer-for-dhs/src/main/java/com/marklogic/hub/dhs/installer/command/InstallIntoDhsCommand.java:80]: Good catch, updated.

**Review by srinathgit — APPROVED** [review]

**Review by rjrudin — APPROVED** [review]

**Review by ryanjdew — APPROVED** [review]

**Pull request merged**